### PR TITLE
New comment style (WIP)

### PR DESCRIPTION
**Pull request by aerojun**
The gravatar should be using the MD5 value of the User, but i don't know how to access it from the comments.

### CODE REVIEW

**Comment by aerojun** [comment]
Don't merge yet.

**Comment by tomleb** [comment]
Could you fix this ? [https://my.mixtape.moe/iafguh.png](https://my.mixtape.moe/iafguh.png)

When clicking a comments id, it goes under the navbar, should go right under it. 

(Some html/css black magic that I don't recall)

**Comment by tomleb** [comment]
Running from your patch I'm getting these weird colors for the advanced search [https://my.mixtape.moe/gffbzh.png](https://my.mixtape.moe/gffbzh.png)

File: templates/view.html [hunk 1]
```diff
@@ -61,49 +60,71 @@ <h3 style="word-break:break-all" {{if eq .Status 2}}class="remake" {{end}} {{if
...
-                            <a href="#comment_{{$index}}">{{$index}}</a>
-                            <a href="{{$.URL.Parse (printf "/user/%d/-" .UserID ) }}">{{.Username}}</a>
+                <div class="commentList">
+                    <h4>{{T "comments"}}</h4>
+                    <hr/>
+                    <ul class="comments">
+                        {{ range $index, $element := .Comments }}
+                        <li class="clearfix comment" id="comment_{{$index}}">
+                            {{/* The following line is wrong. Should be using the MD5 User hash but i don't knwo how to access it. */}}
+                            <img src="https://www.gravatar.com/avatar/{{ .UserID }}?s=50" class="avatar" alt="">
```
Comment (akuma06) [inline templates/view.html]: {{ .MD5 }}
It is that simple 👍

Comment (tomleb) [inline templates/view.html]: We're dropping support for gravatar though

Comment (aerojun) [inline templates/view.html]: @akuma06 Using {{ .MD5 }} thows the following error:

`template: view.html:90:73: executing "content" at <.MD5>: can't evaluate field MD5 in type model.CommentJSON`

**Comment by aerojun** [comment]
Changed the colors an design:

http://i.imgur.com/SEldguS.png
http://i.imgur.com/zGIyghE.png

**Comment by aerojun** [comment]
I've fixed the anchor issue @tomleb. I'll make a PR later today.

**Comment by ewhal** [comment]
@aerojun is this ready?

**Comment by akuma06** [comment]
Resolved conflicts

**Comment by aerojun** [comment]
It was ready 2 days ago.

**Pull request merged**